src/index.js

```diff
@@ -0,0 +1,127 @@
+'use babel';
+
+// eslint-disable-next-line import/extensions, import/no-extraneous-dependencies
+import { CompositeDisposable } from 'atom';
+import path from 'path';
+import escapeHtml from 'escape-html';
+import pluralize from 'pluralize';
+import * as helpers from 'atom-linter';
+
+const DEFAULT_ARGS = [
+  '--cache', 'false',
+  '--force-exclusion',
+  '--format', 'json',
+  '--display-style-guide',
+];
+
+const formatMessage = ({ message, cop_name: copName, url }) => {
+  const formattedMessage = escapeHtml(message || 'Unknown Error');
+  let formattedCopName = '';
+  if (copName && url) {
+    formattedCopName = ` (<a href="${escapeHtml(url)}">${escapeHtml(copName)}</a>)`;
+  } else if (copName) {
+    formattedCopName = ` (${escapeHtml(copName)})`;
+  }
+  return formattedMessage + formattedCopName;
+};
+
+const parseFromStd = (stdout, stderr) => {
+  let parsed;
+  try {
+    parsed = JSON.parse(stdout);
+  } catch (error) {
+    // continue regardless of error
+  }
+  if (typeof parsed !== 'object') { throw new Error(stderr || stdout); }
+  return parsed;
+};
+
+const forwardRubocopToLinter =
+  ({ message: rawMessage, location, severity, cop_name }, filePath) => {
+    const [message, url] = rawMessage.split(/ \((.*)\)/, 2);
+    const { line, column, length } = location || { line: 1, column: 1, length: 0 };
+    return {
+      type: ['refactor', 'convention', 'warning'].includes(severity) ? 'Warning' : 'Error',
+      html: formatMessage({ cop_name, message, url }),
+      filePath,
+      range: [[line - 1, column - 1], [line - 1, (column + length) - 1]],
+    };
+  };
+
+export default {
+  activate() {
+    this.subscriptions = new CompositeDisposable();
+
+    // Register fix command
+    this.subscriptions.add(
+      atom.commands.add('atom-text-editor', {
+        'linter-rubocop:fix-file': async () => {
+          const textEditor = atom.workspace.getActiveTextEditor();
+          const filePath = textEditor.getPath();
+          const command = this.command
+                              .split(/\s+/)
+                              .filter(i => i)
+                              .concat(DEFAULT_ARGS, '--auto-correct', filePath);
+
+          if (!atom.workspace.isTextEditor(textEditor) || textEditor.isModified()) {
+            // Abort for invalid or unsaved text editors
+            return atom.notifications.addError('Linter-Rubocop: Please save before fixing');
+          }
+
+          const cwd = path.dirname(helpers.find(filePath, '.'));
```

Comment:
I still don't see the reasoning behind this use of `helpers.find`. Why isn't this just `path.dirname(filePath)`?